shell/platform/windows/flutter_windows_engine.h

```diff
@@ -269,6 +269,10 @@ class FlutterWindowsEngine {
   // Called when a WM_DWMCOMPOSITIONCHANGED message is received.
   void OnDwmCompositionChanged();
 
+  // Called in response to the framework registering a ServiceBindings.
+  // Registers the top level handler for the WM_CLOSE window message.
+  void OnServiceBindingsRegistered();
```

Comment:
Don't forget to update these names to reflect the latest framework message name :)